### PR TITLE
chore: update copyrights

### PR DESCRIPTION
**Pull request by theseion**

### CODE REVIEW

**Comment by dune73** [comment]
We need to sort out the new integration tests first.

**Comment by azurit** [comment]
@dune73 Plugin needs LuaSocket library so it needs to be installed into testing envirnoment.

**Comment by dune73** [comment]
I see. I think implementing that would be worthwhile since there are more plugins using lua, and that's the direction for plugins anyways.

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 the OWASP Core Rule Set contributors
+   Copyright  contributors
```
Comment (fzipi) [inline LICENSE:189]: Broken replace? All over the place I guess...

Comment (dune73) [reply LICENSE:189]: But this one here is a bit troubling indeed.

Comment (theseion) [reply LICENSE]: fixed

**Review by fzipi — APPROVED** [review]

**Pull request merged**